packages/engine/src/avatar/functions/avatarFunctions.ts

```diff
@@ -314,39 +328,45 @@ export function makeSkinnedMeshFromBoneData(bonesData): SkinnedMesh {
   return skinnedMesh
 }
 
+/**
+ * Find the head bone
+ * @param root
+ */
+
+function findHeadBone(root: Object3D) {
```

Comment:
Probably it's better to move this function into `AvatarBoneMatching.ts`. There is also a similar function there called `_findHead`.